src/bitcoinrpc.cpp

```diff
@@ -437,7 +438,9 @@ static const CRPCCommand vRPCCommands[] =
     { "showblock",               &showblock,               true,   cat_network       },
     { "stop",                    &stop,                    true,   cat_network       },
     { "vote",                    &vote,                    false,  cat_network       },
-    { "votedetails",             &votedetails,             true,   cat_network       }
+    { "votedetails",             &votedetails,             true,   cat_network       },
+    { "getsupervotes",          &rpc_getsupervotes,        false,  cat_developer     },
+    { "getrecentblocks",         &rpc_getrecentblocks,     false,  cat_developer     },
```

Comment:
I noticed some ordering in this file. Is it ok to put my commands simply at the end? @Foggyx420

Reply:
kept it ordered to be simple but u can do as u wish. i'm sure a future cleanup will have it moved as I plan to remove safemode once ur rpc commands are put in since it is removed from bitcoin as well and not used